setup.sh

```diff
@@ -122,101 +156,131 @@ if [[ -z "$SKIP_SEEDING" ]]; then
     fi
     approver="${approver:-$account}"
 
-    gcloud builds submit content-api/data  --project="$OPS_PROJECT" \
+    gcloud builds submit content-api/data  --project="$OPS_PROJECT" --async \
     --substitutions=_FIREBASE_PROJECT="${STAGE_PROJECT}",_APPROVER_EMAIL="${approver}" \
     --config=./content-api/data/cloudbuild.yaml
 
     if [ "${PROD_PROJECT}" != "${STAGE_PROJECT}" ]; then
-        gcloud builds submit content-api/data  --project="$OPS_PROJECT" \
+        gcloud builds submit content-api/data  --project="$OPS_PROJECT" --async \
         --substitutions=_FIREBASE_PROJECT="${PROD_PROJECT}",_APPROVER_EMAIL="${approver}" \
         --config=./content-api/data/cloudbuild.yaml
     fi
 fi # skip seeding
 
-####################
-# Build Containers #
-####################
-
-SETUP_IMAGE_TAG="setup"
-E2E_RUNNER_TAG="latest"
-
-if [[ -z "$SKIP_BUILD" ]]; then
-
-echo
-echo "$(tput bold)Building container images for testing and application hosting...$(tput sgr0)"
-echo
-
-gcloud builds submit "content-api" \
-    --config=ops/api-build.cloudbuild.yaml \
-    --project="$OPS_PROJECT" --substitutions=_REGION="$REGION",_IMAGE_TAG="$SETUP_IMAGE_TAG",_CONTEXT="."
-
-gcloud builds submit \
-    --config=ops/web-build.cloudbuild.yaml \
-    --ignore-file=ops/web-build.gcloudignore \
-    --project="$OPS_PROJECT" --substitutions=_REGION="$REGION",_IMAGE_TAG="$SETUP_IMAGE_TAG"
-
-gcloud builds submit "ops/e2e-runner" \
-    --config=ops/e2e-runner-build.cloudbuild.yaml \
-    --project="$OPS_PROJECT" --substitutions=_REGION="$REGION",_IMAGE_TAG="$E2E_RUNNER_TAG"
-
-fi # skip build
-
-
 ##################
 # Deploy Services #
 ##################
 
+# Function will check build before deploying Run service
+check_for_build_then_run () {
+    local build_id="${1}"
+    local run_command="${2}"
+    # Wait for build to complete.
+    if [ $(gcloud builds describe $build_id --format='value(status)') == "WORKING" ]; then
+        log_url=$(gcloud builds describe $build_id --format='value(logUrl)')
+        echo "Build $build_id still working."
+        echo "Logs are available at [ $log_url ]."
+        while [ $(gcloud builds describe $build_id --format='value(status)') == "WORKING" ]
+        do
+          echo "Build $build_id still working..."
+          sleep 10
+        done
+    fi
+    # Return error and build log for failures. 
+    if [ $(gcloud builds describe $build_id --format='value(status)') == "FAILURE" ]; then
+        build_describe=$(gcloud builds describe $build_id --format='csv(failureInfo.detail,logUrl)[no-heading]')
+        fail_info=$(echo $build_describe | awk -F',' '{gsub(/""/,"\"");print $1}')
```

Comment:
suggestion: It's helpful to have some more comments around this level of bash parsing for folks newer to shell scripting.